lib/inputs/rpm/static.ts

```diff
@@ -14,62 +14,21 @@ export const getRpmDbFileContentAction: ExtractAction = {
 
 export async function getRpmDbFileContent(
   extractedLayers: ExtractedLayers,
-  tmpDirPath?: string,
 ): Promise<string> {
   const rpmDb = getContentAsBuffer(extractedLayers, getRpmDbFileContentAction);
   if (!rpmDb) {
     return "";
   }
 
-  const filePath = generateTempFileName(tmpDirPath);
-  await writeToFile(filePath, rpmDb);
-
   try {
-    // This is the tool that is expected to be found on the system:
-    // https://github.com/snyk/go-rpmdb
-    const cmdOutput = await execute("rpmdb", ["-f", filePath]).catch(
-      handleError,
-    );
-    return cmdOutput.stdout || "";
-  } finally {
-    await removeFile(filePath);
-  }
-}
-
-function handleError(error): CmdOutput | never {
-  const stderr = error.stderr;
-  if (typeof stderr === "string" && stderr.indexOf("not found") >= 0) {
-    return { stdout: "", stderr: "" };
+    const parserResponse = await getPackages(rpmDb);
```

Comment:
why do we need the try/catch here if we're just printing the error and returning an empty string?
in case ```getPackages``` throws even though theoretically it shouldn't?

Reply:
Correct, it's in case`getPackages` throws, if the DB file is bad/corrupted or some of our assumptions about the DB have changed.